stable/prometheus-consul-exporter/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "prometheus-consul-exporter.fullname" . }}
+  labels:
+    app: {{ template "prometheus-consul-exporter.name" . }}
+    chart: {{ template "prometheus-consul-exporter.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  annotations:
```

Comment:
See https://github.com/helm/charts/pull/9128
I just learned today that this is best parameterized, for instance the annotations could be in values, so that a user can override (e.g. remove) whatever is not applicable for her/his environment.